plugins/node/opentelemetry-instrumentation-express/test/hooks.test.ts

```diff
@@ -179,7 +179,7 @@ describe('ExpressInstrumentation hooks', () => {
 
     it('should call requestHook when set in config', async () => {
       const requestHook = sinon.spy((span: Span, info: ExpressRequestInfo) => {
-        span.setAttribute(SEMATTRS_HTTP_METHOD, info.request.method);
+        span.setAttribute(ATTR_HTTP_REQUEST_METHOD, info.request.method);
```

Comment:
Review note: Yes, this is changing from old HTTP semconv `http.method` to new stable HTTP semconv `http.request.method`. However, this is not testing that instr-express is generating this. It is just testing that the `requestHook` is being called.  We could satisfy the test by using `'my.http.method'` or `'foo'` or whatever.